src/ocean/util/serialize/contiguous/Deserializer.d

```diff
@@ -397,7 +397,7 @@ struct Deserializer
     {
         debug (DeserializationTrace)
         {
-            Stdout.formatln("< countRequiredSize!({})({}) : {}\n", S.stringof,
+            Stdout.formatln("< countRequiredSize!({})({}) : {}", S.stringof,
```

Comment:
This looks like it should be in the "Fix function name in DeserializationTrace " commit.